packages/analytics/amplify_analytics_pinpoint_dart/lib/src/impl/analytics_client/event_client/event_client.dart

```diff
@@ -107,6 +107,8 @@ class EventClient implements Closeable {
     return completer.future;
   }
 
+  final _numFailuresByEvent = <String, int>{};
```

Comment:
*nit:* Easier to understand without the abbreviation
```suggestion
  final _failureCountByEvent = <String, int>{};
```